types/documentdb/index.d.ts

```diff
@@ -182,70 +181,319 @@ export interface AuthOptions {
 export interface Procedure extends UniqueId {
 
     /** The function representing the stored procedure. */
-    body(...params: any[]): void;
+    body(...params: any[]): void | string;
```

Comment:
This is from an older commit. Find `UserFunction` in the latest commit to see the updated definition.

The correct form should show a member `body` that is either a function or a string.